exercises/practice/isogram/.approaches/bitfield/content.md

```diff
@@ -0,0 +1,63 @@
+# Bit field
```

Comment:
Love this solution, as it is quite unconventional for C# programmers to write it like this (which makes it interesting)